### DISCUSSION TITLE
Is there a way to log Chat messages?

### DISCUSSION THREAD

**Discussion by marius-smg** [FlowiseAI/Flowise#529] [Q&A]
Hey, i am looking for a way to log the messages user send to my chatbots so i can analyze what kind of question they ask and how much  my chatbots are used. Is there any way i can do this right now? Thanks for any help!

**Comment by chungyau97** [comment]
Hi @marius-smg,

You can achieve that easily using [Zapier Zaps](https://zapier.com/app/dashboard)

Implementation [docs](https://docs.flowiseai.com/3rd-party-platform-integration/zapier-zaps)

**Reply by atthedanieljuma** [reply]
Would you mind showing me how to do this? I can't see how to implement this using Zapier Zaps @chungyau97 @marius-smg 
Thanks

**Reply by marius-smg** [reply]
Hey, i have no idea as well. There is a Log Feature within flowise but i dont know where the Logifiles are stored when i host it on render.com

**Comment by chungyau97** [comment]
Hi @atthedanieljuma,

![image](https://github.com/FlowiseAI/Flowise/assets/33013947/bef75226-5ec9-468c-832e-3e4be9b516e9)

This example get new message from Discord to generate AI output from FlowiseAI
After you have a FlowiseAI's output; you can chain them to another actions to generate output in other platform shown below

![image](https://github.com/FlowiseAI/Flowise/assets/33013947/dbc42002-e294-4bc7-9048-4727722d4ff7)
![image](https://github.com/FlowiseAI/Flowise/assets/33013947/c99db502-c537-4b8f-9b2b-a915553b56af)

Reference:
[FlowiseAI Zapier Zaps](https://docs.flowiseai.com/3rd-party-platform-integration/zapier-zaps)

**Comment by chungyau97** [comment]
Hi @marius-smg,

Please refer here #689
